src/mongo/db/pipeline/document_source_skip_test.cpp

```diff
@@ -65,5 +65,28 @@ TEST_F(DocumentSourceSkipTest, ShouldPropagatePauses) {
     ASSERT_TRUE(skip->getNext().isEOF());
 }
 
+TEST_F(DocumentSourceSkipTest, SkipsChainedTogetherShouldNotOverFlowWhenOptimizing) {
+    // $skip should not optimize if overflow
```

Comment:
'if overflow' doesn't really make sense. How about 'if combining the two values of skips would overflow a long long'. Ditto for the comment below.